e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/filecreationissue/FileCreationIssueEMTest.java

```diff
@@ -0,0 +1,174 @@
+package org.evomaster.e2etests.spring.examples.filecreationissue;
+
+import com.foo.rest.examples.spring.endpointfilter.EndpointFilterController;
+import com.foo.rest.examples.spring.endpointfocusandprefix.EndpointFocusAndPrefixController;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FileCreationIssueEMTest extends SpringTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+
+        SpringTestBase.initClass(new EndpointFocusAndPrefixController());
+    }
+
+    /**
+     * If executive summary is requested, the name of the file should be
+     * EvoMaster_fault_representatives_Test.java, there should not be a file
+     * with name EvoMaster_Test.java
+     * @throws Throwable
+     */
+    @Test
+    public void testFileCreationIssueWithSummary() throws Throwable {
+
+        File outFolder = null;
+
+        try {
+
+            String outFolderName = "fileCreationIssueTest";
+
+            List args = new ArrayList<String>();
+
+            args.add("--blackBox");
+            args.add("true");
+            args.add("--bbTargetUrl");
+            args.add(baseUrlOfSut);
+            args.add("--bbSwaggerUrl");
+            args.add(baseUrlOfSut + "/v2/api-docs");
+            args.add("--outputFormat");
+            args.add("JAVA_JUNIT_4");
+            args.add("--outputFolder");
+            args.add(outFolderName);
+            args.add("--executiveSummary");
+            args.add("true");
+            args.add("--maxTime");
+            args.add("30s");
```

Comment:
that's quite an high value... that ll make the test run for at least 30s... is it really necessary?